.editorconfig

```diff
@@ -253,6 +253,11 @@ dotnet_diagnostic.IDE0060.severity = warning
 # IDE0060: Remove unused parameter
 dotnet_diagnostic.IDE0060.severity = none
 
+[src/Compilers/**/*.{cs,vb}]
+
+# CA1052: Static holder types should be Static or NotInheritable
+dotnet_diagnostic.CA1052.severity = none
```

Comment:
Let's put this in `src/Compilers/.editorconfig` instead.

Reply:
Done